bin/setup

```diff
@@ -8,10 +8,14 @@ chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   step "Installing dependencies" do
-    system! "gem install bundler --conservative"
+    system!("gem install bundler --conservative")
     system("bundle check") || system!("bundle install")
   end
 
+  step "Installing foreman for controlling local servers" do
+    system!("gem install foreman")
```

Comment:
should we check if foreman is already installed before doing this?

Reply:
What's the down-side to placing it in the Gemfile explicitly?

Reply:
https://github.com/thoughtbot/suspenders/pull/330 :)

Reply:
I tend to lean in the direction of "Things that live completely outside the application should not be in the Gemfile"  (Say, `rerun` or `mailcatcher`) but have no strong preferences here.

Reply:
> should we check if foreman is already installed before doing this?
 
I thought of this, but if we call `gem install` and it **is** installed, bundler will just upgrade it. So I'm inclined to leave it, unless you feel like that is an unpleasant surprise.